protocol/src/traits/binding.rs

```diff
@@ -234,27 +234,27 @@ pub trait StoreUint64 {
 
     // Add val with self
     // And set the result back to self
-    fn add(&mut self, val: u64);
+    fn add(&mut self, val: u64) -> bool;
```

Comment:
How about name the method `fn safe_xx` ?
Because we had took care of math overflow.

Reply:
i think right